contracts/core/liquidators/twap-impl/TWAPAuction.sol

```diff
@@ -39,4 +56,357 @@ contract TWAPAuction is TwoAssetPriceBoundedLinearAuction {
         uint256 chunkAuctionPeriod;
         uint256 chunkSize;
     }
+
+    struct TWAPLiquidatorData {
+        uint256 usdChunkSize;
+        uint256 chunkAuctionPeriod;
+    }
+
+    /* ============ Constants ============ */
+
+    // Auction completion buffer assumes completion potentially 2% after fair value when auction started
+    uint256 constant AUCTION_COMPLETION_BUFFER = 2;
+
+    /* ============ State Variables ============ */
+
+    mapping(bytes32 => BoundsLibrary.Bounds) public chunkSizeWhiteList;
```

Comment:
Would define each of these variables w/ a comment.

chunkSizeWhiteList: `Mapping between an address pair's unique keccak256 hash and the min/max USD-chunk size`

expectedChunkAuctionLength: `Estimated length in seconds of a chunk auction`